middleware/apimiddleware_test.go

```diff
@@ -81,13 +86,6 @@ func TestGetAddressCtx(t *testing.T) {
 			wantErr:  true,
 			errMsg:   "invalid address 'DcxxxxcGjmENx4DhNqDctW5wJCVyT3Qeqkx' for this network: checksum mismatch",
 		},
-		{
-			testName: "not_set",
-			args:     args{2, nil},
-			want:     nil,
-			wantErr:  true,
-			errMsg:   "address not set",
-		},
```

Comment:
This was not very useful because the URL reduced to "/address/", which would be a be a different handler so chi was just returning a 404.